cloud-config.txt

```diff
@@ -0,0 +1,61 @@
+#cloud-config
+
+#v1.0
+#Created two user accounts for the server admins. Automatically adds public keys for connection and provides passwordless
+#sudo access. The configuration also edits the sshd_config file to prevent root ssh access and allows ssh access from the two # users
```

Comment:
can you update these docs and move them to the README, along with some usage instructions pls?

Reply:
Done friend